src/herder/Herder.h

```diff
@@ -141,8 +141,8 @@ class Herder
     {
     }
 
-    virtual void dumpInfo(Json::Value& ret, size_t limit) = 0;
-    virtual void dumpQuorumInfo(Json::Value& ret, NodeID const& id,
-                                bool summary, uint64 index = 0) = 0;
+    virtual Json::Value getJsonInfo(size_t limit) = 0;
```

Comment:
I am not sure about this change: `Json::Value` doesn't have a move constructor, so standardizing on this pattern doesn't seem that much better

Reply:
I think that NRVO helps us here (and also fact that those are called by HTTP endpoint which is not used very frequently and does not look like performance critical code path).